src/views/Delegate.vue

```diff
@@ -199,6 +264,32 @@ onMounted(async () => {
             />
           </div>
         </Block>
+        <Block
+          v-if="
+            delegatesLoading || spaceLoading || delegatesWithScore.length > 0
+          "
+          :title="$t('delegate.topDelegates')"
+          :slim="true"
+          :loading="delegatesLoading || spaceLoading"
+        >
+          <div
+            v-for="(delegate, i) in delegatesWithScore"
+            :key="i"
```

Comment:
isnt using index as keys dangerous?

Reply:
Why? This is common practice in Vue

Reply:
It is actually bad to use the index as a key in ReactJS, 

For Vue, I found this https://stackoverflow.com/a/44531545/5916893 maybe something changed in Vue3?
also `delegatesWithScore` won't be changing like a form right just displaying what we get, so maybe it should be fine in this case

Reply:
Oh yes right, I was confusing Index with Item, we should use delegate as the key here.